src/views/StepSunshineView.vue

```diff
@@ -7,6 +7,8 @@ import { useSimulationStore } from '@/stores/simulations'
 import SimulationSteps from '@/components/simulation/SimulationSteps.vue'
 import StepTitle from '@/components/simulation/StepTitle.vue'
 import StepDescription from '@/components/simulation/StepDescription.vue'
+import CustomContent from '@/components/simulation/CustomContent.vue'
```

Comment:
The name of CustomContent is  not explicit enough
You can modify the name of component by "ExplanationSelectionObstacles" for example